src/languages/diff.js

```diff
@@ -17,7 +17,7 @@ export default function(hljs) {
         className: 'meta',
         relevance: 10,
         match: regex.either(
-          /^@@ +-\d+,\d+ +\+\d+,\d+ +@@/,
+          /^@@ +-\d+(?:,\d+)? +\+\d+(?:,\d+)? +@@/,
```

Comment:
Lets keep this simple by keeping the original regex line and then adding a new one for the single number case.

Reply:
We would actually need three new lines for a total of four if we did it that way, since `-1 +1,2` and `-1,2 +1` are also both valid:

```
/^@@ +-\d+,\d+ +\+\d+,\d+ +@@/
/^@@ +-\d+ +\+\d+,\d+ +@@/
/^@@ +-\d+,\d+ +\+\d+ +@@/
/^@@ +-\d+ +\+\d+ +@@/
```

IMHO this does not end up simpler to read, but I don't mind changing it if you prefer, just let me know!

I'll also add test coverage for those other cases, which is probably a good idea in any case.

Reply:
> I'll also add test coverage for those other cases, which is probably a good idea in any case.

Yes, awesome.

> IMHO this does not end up simpler to read,

Yep, matter of opinion.  What might help further though is put an example in a comment after each of the regex.  That should make it 100% clear I think for all readers.

Reply:
Sounds good, done.